storm-server/src/main/java/org/apache/storm/daemon/supervisor/Supervisor.java

```diff
@@ -327,6 +330,15 @@ public void launch() throws Exception {
         LOG.info("Starting supervisor with id {} at host {}.", getId(), getHostName());
     }
 
+    private void validateConf() {
+        List<Number> ports = (List<Number>) conf.get(DaemonConfig.SUPERVISOR_SLOTS_PORTS);
+        Set<Integer> numaPorts = SupervisorUtils.getNumaPorts(conf);
+        numaPorts.removeAll(ports);
+        if (numaPorts.size() > 0) {
```

Comment:
Lately, we added NUMA support, so if getNumaPorts is configured and not empty, numaPorts should be used otherwise it can default to slots_ports. Since we are providing more than one place to configure ports, I think it is implied that NUMA one is newer and should win if configured. No need to stop supervisor daemon.

Reply:
To avoid user confusion, we should update Java Documentation on configuration suggesting that if numa_ports are configured they take precedence over slots.ports.

Reply:
Updated comment in https://issues.apache.org/jira/browse/STORM-3615 to address as part of the NUMA feature documentation.

Reply:
@kishorvpatil - updated to add numa ports to slots ports config.